snapcraft/tests/test_project_loader.py

```diff
@@ -885,6 +885,30 @@ def test_invalid_environment(self, mock_load_plugin):
             "The 'environment/INVALID' property does not match the required "
             "schema: \[1, 2\].*")
 
+    @unittest.mock.patch('snapcraft.internal.parts.PartsConfig.load_plugin')
+    def test_invalid_yaml_empty_version(self, mock_loadPlugin):
+        fake_logger = fixtures.FakeLogger(level=logging.ERROR)
+        self.useFixture(fake_logger)
+
+        self.make_snapcraft_yaml("""name: test
+version:''
+summary: test
+description: test
+confinement: strict
+grade: stable
+parts:
+  part1:
+    plugin: go
+    stage-packages: [fswebcam]
```

Comment:
The idea is to keep the test as small as possible, so you can save one line by using `plugin: nil` and removing `stage-packages`.